01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html class="html">
+<head>
+   <link href="normalize.css" rel="stylesheet">
+   <link href="exerciseJS.css" rel="stylesheet">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <meta charset="UTF-8">
+   <title>Excercise 2</title>
+   <script type="text/javascript" src="exerciseJS.js"> </script>
+   </head>
```

Comment:
I checked and corrected every indentation problem, but now i'm not sure if it's right or i'm wrong about indentation standar.